tsconfig.base.json

```
@@ -36,6 +33,7 @@
       ],
       "@code-pushup/models": ["packages/models/src/index.ts"],
       "@code-pushup/nx-plugin": ["packages/nx-plugin/src/index.ts"],
+      "@code-pushup/plugin-jsdocs": ["packages/plugin-jsdocs/src/index.ts"],
```

The TS path alias should match the `package.json` name:

```suggestion
      "@code-pushup/jsdocs-plugin": ["packages/plugin-jsdocs/src/index.ts"],
```